packages/material-ui/src/Rating/Rating.js

```diff
@@ -442,21 +533,20 @@ const Rating = React.forwardRef(function Rating(props, ref) {
         });
       })}
       {!readOnly && !disabled && valueRounded == null && (
-        <label className={clsx({ [classes.labelEmptyValueActive]: emptyValueFocused })}>
-          <input
+        <RatingLabel styleProps={styleProps}>
+          <RatingVisuallyHiddenInput
```

Comment:
I am positive that we should have one `RatingVisuallyHidden` component, or even better we can add in the root styles with sub-selector for the visually hidden styles and just apply `className` on these primitive HTML elements. That can help us have better perf too. Something like:

```
  `& .${classes.visallyHidden}`: {
    ...visuallyHiddem,
  }
```
and apply the `classes.visuallyHidden` on the HTML elements direclty.